src/healthkit.cs

```diff
@@ -2442,4 +2566,225 @@ interface HKWorkoutRouteQuery {
 		IntPtr Constructor (HKWorkoutRoute workoutRoute, HKWorkoutRouteBuilderDataHandler dataHandler);
 	}
 
+	delegate void HKWorkoutBuilderCompletionHandler (bool success, NSError error);
+	[Watch (5,0), iOS (12,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface HKWorkoutBuilder
+	{
+		[NullAllowed, Export ("device", ArgumentSemantic.Copy)]
+		HKDevice Device { get; }
+
+		[NullAllowed, Export ("startDate", ArgumentSemantic.Copy)]
+		NSDate StartDate { get; }
+
+		[NullAllowed, Export ("endDate", ArgumentSemantic.Copy)]
+		NSDate EndDate { get; }
+
+		[Export ("workoutConfiguration", ArgumentSemantic.Copy)]
+		HKWorkoutConfiguration WorkoutConfiguration { get; }
+
+		[Protected]
+		[Export ("metadata", ArgumentSemantic.Copy)]
+		NSDictionary _Metadata { get; }
```

Comment:
API with leading underscores should not be exposed (even if it's protected it's still exposed). Maybe something like `NativeMetadata` instead? IIRC we've used a `Native` suffix for something like this elsewhere.